java/build.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project name="java" basedir="../nbbuild" default="build">
+    <description>NetBeans Java cluster  build script.</description>
+    <import file="../nbbuild/build.xml"/>
+
+  <target name="build" description="Build the NetBeans Java cluster.">
+      <mkdir dir="${netbeans.dest.dir}"/>
+      <ant dir="." target="build" inheritAll="false">
+          <property name="cluster.config" value="standard-java"/>
+          <property name="permit.jdk9.builds" value="true"/>
+      </ant>
+  </target>
+
+ <target name="test" description="Test the NetBeans Java modules.">
+      <mkdir dir="${netbeans.dest.dir}"/>
+      <ant dir="." target="test" inheritAll="false">
+          <property name="cluster.config" value="java"/>
```

Comment:
The value should be consistent with the value on line 29.

Reply:
Had added new axis for basic cluster in .travis.yml. Already there are entries in .travis.yml to run test cases for few java modules, so I haven't added any new entry for running test-cases in .travis.yml file